_data/translation-management-systems.yaml

```diff
@@ -0,0 +1,669 @@
+---
+#- id: [Lowercase, words separated by hyphen]
+#  name:
+#  description:
+#  url:
+#  cat: [Boolean]
```

Comment:
if this is always true, maybe we don't need it?

Reply:
I've changed `cat` to `type`, and made it a list, where values are either `cat`, `tms`, or both.
An alternative could be that, instead of having `cat: [boolean]`, we can have `tms: [boolean]` as this is the characteristic that changes.

Reply:
Then it needs to always be a list.

`types: [ ... ]`

But I would lean towards the boolean approach.